ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -56,7 +56,8 @@ const PostHeaderBanner = ({
 }) => {
   const {
     timezone,
-    locale,
+    datetimeLocale,
+    serviceDatetimeLocale,
```

Comment:
I searched the repo for services which have set `serviceDatetimeLocale` - ukrainian is the only one which overrides the value - could we add a test for it to both the relative time + `Shows timestamp in the expected format for $service for articles over 10 hours old.` test above?